packages/renderer/src/lib/markdown/micromark-image-directive.ts

```
@@ -0,0 +1,86 @@
+ * - width: image width (optional)
+ * - height: image height (optional)
+ * - class: additional CSS classes (optional)
+ */
+/**
+ * @this {import('micromark-util-types').CompileContext}
+ * @type {import('micromark-extension-directive').Handle}
+ */
+export function image(d: Directive): void {
+  // Make sure it's not part of a text directive
+  if (d.type !== 'textDirective') {
+    return false;
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)